cypress/support/commands/testUser.ts

```diff
@@ -253,7 +253,7 @@ export const createTestUser = ({
 			})
 			.then((res) => {
 				return cy.wrap({
-					emailAddress: finalEmail,
+					emailAddress: res.body.emailAddress as string,
```

Comment:
this field is added here
https://github.com/guardian/identity/pull/2528/files#diff-dc9b5164ef33bc1fce5db7e899dbb88e983e78c983e09c566eca504b66445efbR37